platform/flux/flux.yaml

```diff
@@ -31,4 +31,4 @@ spec:
       path: "platform"
       secretName: flux-key
     additionalArgs:
-      - --k8s-allow-namespace=control,delivery,security,logging,monitoring,tracing,kube-system,flux,istio-system
\ No newline at end of file
+      - --k8s-allow-namespace=control,delivery,security,logging,monitoring,kube-system,flux,istio-system
```

Comment:
Ensure that ingresses are created in namespace `istio-system` instead of `tracing`